packages/react-native-session-replay/android/src/main/kotlin/com/datadog/reactnative/sessionreplay/mappers/ReactTextMapper.kt

```diff
@@ -0,0 +1,57 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.reactnative.sessionreplay.mappers
+
+import android.widget.TextView
+import androidx.annotation.VisibleForTesting
+import com.datadog.android.sessionreplay.internal.AsyncJobStatusCallback
+import com.datadog.android.sessionreplay.internal.recorder.MappingContext
+import com.datadog.android.sessionreplay.internal.recorder.mapper.TextViewMapper
+import com.datadog.android.sessionreplay.model.MobileSegment
+import com.datadog.reactnative.sessionreplay.ReactTextPropertiesResolver
+import com.facebook.react.bridge.ReactContext
+import com.facebook.react.uimanager.UIManagerModule
+
+internal class ReactTextMapper(
+    private val reactContext: ReactContext,
+    private val uiManagerModule: UIManagerModule?,
+    private val reactTextPropertiesResolver: ReactTextPropertiesResolver =
+        ReactTextPropertiesResolver(reactContext, uiManagerModule)
+): TextViewMapper() {
+    override fun map(
+        view: TextView,
+        mappingContext: MappingContext,
+        asyncJobStatusCallback: AsyncJobStatusCallback
+    ): List<MobileSegment.Wireframe> {
+        val result: MutableList<MobileSegment.Wireframe> = mutableListOf()
+        val wireframes = mapOnSuperclass(view, mappingContext, asyncJobStatusCallback)
+        val pixelDensity = mappingContext.systemInformation.screenDensity
+
+        wireframes.forEach{ originalWireframe ->
```

Comment:
we need to setup `ktlint` format + check

```suggestion
        wireframes.forEach { originalWireframe ->
```